README.md

```diff
@@ -47,4 +47,15 @@ Build the Next.js Code and deploy
 npm i
 num run build-serverless-next
 terraform apply
+```
+
+### Tests
+This module is tested with terratest, which is essentially a Go test library with cloud and Terraform SDK integrations
+
+To run the tests manually:
+```bash
+cd test/
+go mod init <my_module>
```

Comment:
Why would the user want to create my his module? pls commit a go.mod

Reply:
Added the go.mod